middleware/index.ts

```diff
@@ -305,25 +294,21 @@ function createLogger(requestID?: string): Logger {
 
 const userFn = getUserFn();
 
-export default async function systemFn(message: object): Promise<object> {
+export default async function systemFn(message: any): Promise<any> {
     const payload = message['payload'];
 
-    // Remap headers to a standard JS object
+    // Remap riff headers to a standard JS object
     const headers = message['headers'].toRiffHeaders();
     Object.keys(headers).map((key: string) => {headers[key] = message['headers'].getValue(key)});
 
-    const requestId = headers['ce-id'] || headers['x-request-id'];
+    const requestId = headers['Ce-Id'] || headers['X-Request-Id'];
     const requestLogger = createLogger(requestId);
     try {
-        const middlewareResult = await applySfFxMiddleware(
-            {payload, headers},
-            {},
-            [payload, requestLogger]);
+        const middlewareResult = await applySfFxMiddleware({payload, headers}, requestLogger);
         const result = await userFn(...middlewareResult);
 
-        // If userFn does not have explicit return, it would be undefined, when || with null, it would be null
-        // for Accept header, riff node invoker's application/json marshaller Buffer.from(JSON.stringify(null))
-        return result || null;
+        // Currently, riff doesn't support undefined or null return values
+        return result || '';
```

Comment:
So this is going to result in a change of behavior for functions without return values right?

Reply:
Maybe empty object `{}` would be better, semantically?